datumaro/plugins/velodynepoints_format/format.py

```diff
@@ -0,0 +1,3 @@
+class VelodynePointsPath:
```

Comment:
There should be a license header with MIT license

Reply:
Resolved